src/main/scala/com/datastax/sparkstress/SparkCassandraStress.scala

```diff
@@ -69,7 +69,7 @@ object SparkCassandraStress {
         config.copy(keyspace = arg)
       } text {"Name of the keyspace to use/create"}
 
-      opt[String]('m',"saveMethod") optional() action { (arg,config) =>
+      opt[String]('s',"saveMethod") optional() action { (arg,config) =>
```

Comment:
Changed these option abbreviations to match their description better, and to free up 'm' for termination time in 'minutes'.